### PR TITLE
Bump euclid and heapsize and bump version

### PR DESCRIPTION
**Pull request by Eijebong**
---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1303)

### CODE REVIEW

File: webrender/Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ app_units = "0.4"
 bincode = "1.0.0-alpha6"
 bit-set = "0.4"
 byteorder = "1.0"
-euclid = "0.11.2"
+euclid = ">= 0.11, < 0.14"
```
Comment (kvark) [inline webrender/Cargo.toml]: I'm not 100% sure about this being a non-breaking change. Is this approach established elsewhere for a reference?

Comment (Eijebong) [inline webrender/Cargo.toml]: Well I didn't change any code so unless you're reexporting euclid somewhere I don't think that this is a breaking change.

Comment (Eijebong) [inline webrender/Cargo.toml]: If you want I can force euclid 0.13 and bump to 0.40.0

**Comment by kvark** [comment]
@glennw heads up - this may prevent updating WR for servo until everything is using euclid 0.13
@bors-servo r+

**Pull request merged**